### PR TITLE
Ux/UI changes

### PR DESCRIPTION
**Pull request by Sweekrity-7392**
1) font change -> indie flower to mali
2) Outlinedbutton text color change
3) customizable size added to OutlinedButton
4) scrollbar thickness changed
5) alignment changed
6) text -> autotextsize : to ensure proper fitting of text on language change
7) appbar size added -> preferredSize

### CODE REVIEW

**Comment by skmanish** [comment]
Please resolve conflicts.

**Comment by Sweekrity-7392** [comment]
> Please resolve conflicts.

resolved!

File: lib/BSE-observation/instructions.dart [hunk 1]
```diff
@@ -17,7 +17,7 @@ class instructions extends StatelessWidget {
     return AlertDialog(
       scrollable: true,
       backgroundColor: kDarkButton.withOpacity(0.1),
-      title: AutoSizeText(
+      title: Text(
```
Comment (skmanish) [inline lib/BSE-observation/instructions.dart:20]: Why is this changed back to Text? Was that intended?

Comment (Sweekrity-7392) [reply lib/BSE-observation/instructions.dart:20]: to resolve an error with intrinsic dimensions.
